cli/examples/device-onNameChanged.sequences.json

```diff
@@ -2,21 +2,21 @@
   {
     "at": 5000,
     "event": {
-      "method": "device.onDeviceNameChanged",
+      "method": "device.onNameChanged",
```

Comment:
One last one... this file should really be called "device-onNameChanged.sequence.json" (no 's' / not plural).